tests/net/wifi/configs/testcase.yaml

```diff
@@ -0,0 +1,66 @@
+common:
+  build_only: true
+  tags:
+    - drivers
+    - wifi
+    - net
+  platform_allow:
+    - native_sim
+    - native_sim/native/64
+tests:
+  wifi.build.crypto_default:
+    extra_configs:
+      - CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO=y
+  wifi.build.crypto_alt:
+    extra_configs:
+      - CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_ALT=y
+  wifi.build.crypto_none:
+    extra_configs:
+      - CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_NONE=y
+  # FIXME: The psa crypto does not work atm so commented out
+  #        the build test temporarily.
+  # wifi.build.crypto_psa:
+  #  extra_configs:
+  #    - CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_MBEDTLS_PSA=y
+  wifi.build.crypto_enterprise:
+    extra_configs:
```

Comment:
@jukkar is the guidance to continue to add the extra configuration items to this test case as they get added to Zephyr?

Reply:
It might not be practical to add new config options here. I think we should have these set differently by having a Kconfig option that sets needed values like these, and then just select that one option here.